### PR TITLE
Clarify the overall structure of a naming rule

### PR DESCRIPTION
**Pull request by jasonmalinowski**
If you're not familiar with the overall structure of a rule this page can be a bit confusing. Attempt to illustrate the various parts using the example we already had.

### CODE REVIEW

File: docs/fundamentals/code-analysis/style-rules/naming-rules.md [hunk 1]
```diff
@@ -19,15 +19,15 @@ helpviewer_keywords:
...
 # Naming rules
 
-Naming rules concern the naming of .NET programming language code elements, such as classes, properties, and methods. For example, you can specify that public members must be capitalized or that private fields must begin with `_`.
+Naming rules let you specify and enforce the naming of .NET programming language code elements, such as classes, properties, and methods. For example, you can specify that public members must be capitalized or that private fields must begin with `_`.
 
 A naming rule has three parts:
 
-* The group of symbols it applies to.
-* The naming style to associate with the rule.
+* The _symbol group_ it applies to. In our examples above, that's the "public members" or "private fields" portion of the rule.
```
Comment (gewarren) [inline docs/fundamentals/code-analysis/style-rules/naming-rules.md:26]: On this line and line 27, I'm not sure what the "examples above" refer to.

File: docs/fundamentals/code-analysis/style-rules/naming-rules.md [hunk 2]
```diff
@@ -19,15 +19,15 @@ helpviewer_keywords:
...
+* The _naming style_ to associate with the rule. In our examples above, that's the "must be capitalized" or "must begin with `_`" portion of the rule.
 * The severity for enforcing the convention.
 
-You define naming rules in an EditorConfig file.
+You define naming rules in an EditorConfig file. First, you must define the symbol group and naming style, and give each of them a title. Then you specify the naming rule which links everything together.
```
Comment (gewarren) [inline docs/fundamentals/code-analysis/style-rules/naming-rules.md:30]:
```suggestion
You define naming rules in an EditorConfig file. First, you must specify the symbol group and naming style and give each of them a title. Then you specify the naming rule, which links everything together.
```

File: docs/fundamentals/code-analysis/style-rules/naming-rules.md (hunk 1 above)
Comment (jasonmalinowski) [reply docs/fundamentals/code-analysis/style-rules/naming-rules.md:26]: The example on line 22. Would it help to maybe make those examples a separate list so they stand out?

File: docs/fundamentals/code-analysis/style-rules/naming-rules.md [hunk 3]
```diff
@@ -19,15 +19,15 @@ helpviewer_keywords:
...
 
-Naming rules concern the naming of .NET programming language code elements, such as classes, properties, and methods. For example, you can specify that public members must be capitalized or that private fields must begin with `_`.
+Naming rules let you specify and enforce the naming of .NET programming language code elements, such as classes, properties, and methods. For example, you can specify that public members must be capitalized or that private fields must begin with `_`.
 
 A naming rule has three parts:
 
-* The group of symbols it applies to.
-* The naming style to associate with the rule.
+* The _symbol group_ it applies to. In our examples above, that's the "public members" or "private fields" portion of the rule.
+* The _naming style_ to associate with the rule. In our examples above, that's the "must be capitalized" or "must begin with `_`" portion of the rule.
```
Comment (gewarren) [inline docs/fundamentals/code-analysis/style-rules/naming-rules.md:27]:
```suggestion
* The _naming style_ to associate with the rule, for example, that the name must be capitalized or start with an underscore.
```

Comment (jasonmalinowski) [reply docs/fundamentals/code-analysis/style-rules/naming-rules.md:27]: Ah, that works well, great suggestion.

**Comment by gewarren** [comment]
@jasonmalinowski I resolved the merge conflicts in this file. Can you see if the changes still convey what you were trying to convey? I think this is ready to merge.

**Review by gewarren — APPROVED** [review]

**Pull request merged**